package.json

```diff
@@ -1,10 +1,16 @@
 {
   "scripts": {
     "test": "jest --coverage",
+    "ui-test": "jest test/js --testPathIgnorePatterns=scp-api.test.js",
     "testDebug": "node --inspect-brk node_modules/.bin/jest --runInBand",
     "lint": "eslint app/javascript",
     "fix-styles": "prettier-stylelint --write 'src/**/*.{css,scss}' "
   },
+  "husky": {
+    "hooks": {
+      "pre-push": "npm test"
```

Comment:
curious why we're using 'npm test' here instead of 'yarn test'.  does it matter?